src/current/_includes/v23.2/known-limitations/physical-cluster-replication.md

```diff
@@ -0,0 +1,14 @@
+- Physical cluster replication is supported only on CockroachDB {{ site.data.products.core }} in new v23.2 clusters. That is, clusters that have been upgraded from a previous version of CockroachDB will not support physical cluster replication.
```

Comment:
This is the all limitations added to an include file. 

**NEW content on lines 2 + 14 here.**